Bottom: 12b1e8990b3798c411666542ffc72c4e0a85d9bb
Top:    e24d1893de0f47a1b237a3304a2f3ee216fc0057
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2013-06-05 12:07:58 +0100

Comment out check for GUI HID when considering hidenames

Yes - I want to hide them for a print!


---

diff --git a/src/draw.c b/src/draw.c
index 13d0077..fa066f3 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -183,7 +183,7 @@ DrawStrippedText (hidGC gc, ElementType *Element, int min_width)
 static void
 draw_element_name (ElementType *element)
 {
-  if ((TEST_FLAG (HIDENAMESFLAG, PCB) && gui->gui) ||
+  if ((TEST_FLAG (HIDENAMESFLAG, PCB) /*&& gui->gui*/) ||
       TEST_FLAG (HIDENAMEFLAG, element))
     return;
   ghid_set_lock_effects (Output.fgGC, (AnyObjectType *)element);
